src/main/java/fr/insalyon/creatis/gasw/GaswUtil.java

```diff
@@ -85,7 +86,7 @@ public static Process getProcess(Logger logger, boolean redirectError,
             builder.redirectErrorStream(true);
         }
 
-        builder.environment().put("X509_USER_PROXY", System.getenv("X509_USER_PROXY"));
+        //builder.environment().put("X509_USER_PROXY", System.getenv("X509_USER_PROXY"));
```

Comment:
Is it mandatory to remove it for MoteurLite ?

Reply:
Its not mandatory, but I think we spoke about removing proxy previously (Trello comment - 22nd Sep 2023 )